src/components/slideshow/slideshow.js

```diff
@@ -387,125 +437,115 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
-            elem.classList.remove('hide');
+            element.classList.remove('hide');
 
             var onFinish = function () {
-                focusManager.focus(elem.querySelector('.btnSlideshowPause'));
+                focusManager.focus(element.querySelector('.btnSlideshowPause'));
             };
 
-            if (!elem.animate) {
+            if (!element.animate) {
                 onFinish();
                 return;
             }
 
             requestAnimationFrame(function () {
-
                 var keyframes = [
-                    { transform: 'translate3d(0,' + elem.offsetHeight + 'px,0)', opacity: '.3', offset: 0 },
+                    { transform: 'translate3d(0,' + element.offsetHeight + 'px,0)', opacity: '.3', offset: 0 },
                     { transform: 'translate3d(0,0,0)', opacity: '1', offset: 1 }
                 ];
                 var timing = { duration: 300, iterations: 1, easing: 'ease-out' };
-                elem.animate(keyframes, timing).onfinish = onFinish;
+                element.animate(keyframes, timing).onfinish = onFinish;
             });
         }
 
-        function slideDownToHide(elem) {
-
-            if (elem.classList.contains('hide')) {
+        /**
+         * Hides the OSD by sliding it out of view.
+         * @param {HTMLElement} element - Element containing the OSD.
+         */
+        function slideDownToHide(element) {
+            if (element.classList.contains('hide')) {
                 return;
             }
 
             var onFinish = function () {
-                elem.classList.add('hide');
+                element.classList.add('hide');
                 _osdOpen = false;
             };
 
-            if (!elem.animate) {
+            if (!element.animate) {
                 onFinish();
                 return;
             }
 
             requestAnimationFrame(function () {
-
                 var keyframes = [
                     { transform: 'translate3d(0,0,0)', opacity: '1', offset: 0 },
-                    { transform: 'translate3d(0,' + elem.offsetHeight + 'px,0)', opacity: '.3', offset: 1 }
+                    { transform: 'translate3d(0,' + element.offsetHeight + 'px,0)', opacity: '.3', offset: 1 }
                 ];
                 var timing = { duration: 300, iterations: 1, easing: 'ease-out' };
-                elem.animate(keyframes, timing).onfinish = onFinish;
+                element.animate(keyframes, timing).onfinish = onFinish;
             });
         }
 
-        var lastMouseMoveData;
-
-        function onPointerMove(e) {
-
-            var pointerType = e.pointerType || (layoutManager.mobile ? 'touch' : 'mouse');
+        /**
+         * Shows the OSD when moving the mouse pointer or touching the screen.
+         * @param {Event} event - Pointer movement event.
+         */
+        function onPointerMove(event) {
+            var pointerType = event.pointerType || (layoutManager.mobile ? 'touch' : 'mouse');
```

Comment:
why are we only utilizing `touch` on mobile? there are laptops with touch

Reply:
The mobile layout is the only one currently made with touch in mind. I don't have a way to test touch on regular layouts, so I kept this part identical to what it was before.